examples/pymc3_howto/sampling_compound_step.ipynb

```diff
@@ -12,10 +12,24 @@
     "# Compound Steps in Sampling\n",
```

Comment:
<p>I would sample from the model using both orders of step methods and add the sample stats section here, showing how each step method gets it's own accept (here accept being log p_updated - log p_current, it is already defined here) and how inverting them affects that. Note that that actually forces us to use idata_kwargs instead of coords at model creation, because even if the dims don't change, the coordinates do.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/141/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>I'll get back to this one, once I get the existing code working </p>